build/Rulesets/Roslyn_BuildRules.ruleset

```diff
@@ -106,6 +106,13 @@
   <Rules AnalyzerId="Microsoft.NetCore.Analyzers" RuleNamespace="Microsoft.NetCore.Analyzers.ImmutableCollections">
     <Rule Id="RS0012" Action="Warning" />
   </Rules>
+  <Rules AnalyzerId="Microsoft.VisualStudio.Threading.Analyzers" RuleNamespace="Microsoft.VisualStudio.Threading.Analyzers">
+    <Rule Id="VSTHRD001" Action="None" /> <!-- don't enforce useage of JTF -->
+    <Rule Id="VSTHRD002" Action="None" /> <!-- don't enforce useage of JTF -->
+    <Rule Id="VSTHRD010" Action="None" /> <!-- don't enforce useage of JTF -->
+    <Rule Id="VSTHRD103" Action="None" /> <!-- without flow analysis this rule has a lot of false positives and provides little value -->
+    <Rule Id="VSTHRD200" Action="None" /> <!-- don't enforce that names must end in 'Async' they are either tests or part of a public API -->
```

Comment:
💭 Seems like this should eventually be moved to the non-shipping projects rule set, and suppressions added for anything that shipped in the public API.